server/meshmodel/aws-rds-controller/v1.4.3/v1.0.0/relationships/dbCluster-to-dbClusterParameterGroup.json

```diff
@@ -0,0 +1,56 @@
+{
+    "schemaVersion": "relationships.meshery.io/v1alpha3",
+    "version": "v1.0.0",
+    "kind": "edge",
+    "type": "binding",
+    "subType": "configuration",
```

Comment:
@Jougan-0 is this an existing subtype? What visualization is associated with "configuration"?

Reply:
I don't know how did it took configuration and backup. I'll investigate why did the LLM gave this.